### PR TITLE
accesibility 3

### PR DESCRIPTION
**Pull request by Himani-Patel12**
# Pull request

## Implement

- [ ] Update pull request (title, assignee, labels, projects, implement, measure)
- [ ] Update issue numbers here which will be fixed by this PR | Fixes #
- [ ] Update labels for clarity & PR workflow
- [ ] Add/Update comments in code
- [ ] Self code review as per this [guide](https://google.github.io/eng-practices/review/)
- [ ] Add/Update change logs, docs and release notes

## Measure

- [ ] What: Pull Request Title | Why: PR Detail | How: DIM | When: Timeline | Who: Assignee & Reviewer
- [ ] Review change logs, docs & release notes
- [ ] After completing all subtasks validate and make PR ready to merge

### CODE REVIEW

**Comment by beingminimal** [comment]
@KunalChudasama Please review this

File: individual-projects/2023/Himani-Patel-190770107153/Accesibility 3/index.html [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+        <article>
+            <ul>
+                <li>><a href="#"> B2C and B2B Booking Portal</a></li>
+                <li><a href="#"> Flight / Hotel / Car Booking Engines</a></li>
+                <li><a href="#">CMS Development And Customization</a></li>
+                <li><a href="#"> Website MainTenance</a></li>
+                <li><a href="#"> White Label Travel Portal</a></li>
+                <li><a href="#"> GDS API Integration</a></li>
+            </ul>
+        </article>
```
Comment (KunalChudasama) [inline individual-projects/2023/Himani-Patel-190770107153/Accesibility 3/index.html:47]: No need to use article tag here..and all links would be in the above section.

File: individual-projects/2023/Himani-Patel-190770107153/Accesibility 3/index.html [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+        <section>
+            <img src="https://www.traveltechnologysolution.com/webroot/images/about_us.jpg" alt="graph">
```
Comment (KunalChudasama) [inline individual-projects/2023/Himani-Patel-190770107153/Accesibility 3/index.html:50]: this img would be the part of the above section.

**Review by KunalChudasama — CHANGES_REQUESTED** [review]
Changes Required

**Comment by Himani-Patel12** [comment]
@KunalChudasama I changed it. Please review.

**Review by KunalChudasama — APPROVED** [review]

**Pull request merged**